<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-L3R1A1J" name="Default test(failed)" tests="4" failures="1" timestamp="30 Dec 2021 02:41:36 GMT" time="0.108" errors="0">
  <testcase name="test2" time="0.026" classname="com.mycompanyTest.TestProject.TestDemo">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.005" classname="com.mycompanyTest.TestProject.TestDemo">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.003" classname="com.mycompanyTest.TestProject.TestDemo">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.003" classname="com.mycompanyTest.TestProject.TestDemo">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.mycompanyTest.TestProject.TestDemo.test2(TestDemo.java:19)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- Default test(failed) -->
